front-end-development/front-end-development-quiz.md

```diff
@@ -318,10 +325,17 @@ for (i = 0; i > max; i++) {
 
 #### Q33. How can you rewrite this function using arrow function syntax?
 
+<<<<<<< HEAD
+- [ ] ```let product => (x,y) { <br /> x \* y; <br />}```
+- [x] ```let product = (x,y) => x\*y;```
+- [ ] ```let product => x\*y;```
+- [ ] ```let product = (x,y) -> x\*y;```
+=======
 - [ ] `let product => (x,y) { <br /> x \* y; <br />}`
 - [x] `let product = (x,y) => x\*y;`
 - [ ] `let product => x\*y;`
 - [ ] `let product = (x,y) -> x\*y;`
+>>>>>>> 1c498d81d8a6287e8d3e60e67ba5e8bb9bb19d3c
```

Comment:
```suggestion
<<<<<<< HEAD
- [ ] ```let product => (x,y) { <br /> x \* y; <br />}```
- [x] ```let product = (x,y) => x\*y;```
- [ ] ```let product => x\*y;```
- [ ] ```let product = (x,y) -> x\*y;```
=======
- [ ] `let product => (x,y) { <br /> x \* y; <br />}`
- [x] `let product = (x,y) => x\*y;`
- [ ] `let product => x\*y;`
- [ ] `let product = (x,y) -> x\*y;`
>>>>>>> 1c498d81d8a6287e8d3e60e67ba5e8bb9bb19d3c
```
should be improved